packages/app/src/runner/SpecRunnerContainerOpenMode.vue

```diff
@@ -22,24 +22,21 @@ const specStore = useSpecStore()
 const router = useRouter()
 const route = useRoute()
 
-const { initialized, watchSpec } = useUnifiedRunner()
-
 const specs = computed(() => {
   return props.gql.currentProject?.specs ?? []
 })
 
-watchSpec(specs)
```

Comment:
Not a strong opinion but I like to functions that call `watch` or `watchEffect` in the actual component file to give visibility, or you need to dig through a bunch of other files to find the watcher. It's like when you are trying to track down a `useState` or `useEffect` nested deeply in a hook in React.

I don't mind too much, but something about `watchSpec(specs)` is more readable to me than `useUnifiedRunner(specs)`. The former is more clear that something reactive will happen. Happy to field opinions and not really a big deal either way.

Reply:
I'd prefer adding 'watcher' to the composable's name in some way if more discoverability is desired. Putting the responsibility of activating the watchers on the consuming component just leaves a logic hole where there doesn't need to be one.